editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -50,33 +49,30 @@ describe('React Render Count Tests - ', () => {
 
     expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
       makeTestProjectCodeWithSnippet(
-        `<View style={{ ...props.style }} layout={{ layoutSystem: 'pinSystem' }} data-uid='aaa'>
+        `<View style={{ ...props.style }} data-uid='aaa'>
         <View
-          style={{ backgroundColor: '#DDDDDD', opacity: 0.3 }}
-          layout={{ layoutSystem: 'pinSystem', left: 52, top: 61, width: 256, height: 202 }}
+          style={{ backgroundColor: '#DDDDDD', position: 'absolute', opacity: 0.3, left: 52, top: 61, width: 256, height: 202 }}
           data-uid='bbb'
         />
       </View>`,
       ),
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(310) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(320)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(670) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
It more than doubles?

Reply:
Eni commented above that this is a real regression already on master - https://github.com/concrete-utopia/utopia/pull/1176#issuecomment-829939128

Reply:
Aha!